web/src/modals.ts

```diff
@@ -152,6 +152,10 @@ export function open(
             return;
         }
 
+        if ($(e.target).is("[data-ignore-overlay-click]")) {
+            return;
```

Comment:
Kept it in this format instead of

```
        if (!$(e.target).is("[data-close-on-overlay-click]")) {
            return;
         }
```

in this case, we'll have to add `data-close-on-overlay-click` at all those `.hbs` files where micomodal is used.